### DISCUSSION TITLE
meta key (left/right alts)

### DISCUSSION THREAD

**Discussion by dblum-analog** [nikitabobko/AeroSpace#745] [questions-and-answers]
Is there way to differentiate between the left/right alt keys in the configuration? 

I'm using iterm2 and would like to set the right option key to be `esc+` for usage with tmux/nvim.

**Comment by aleksanderwalczuk** [comment]
I'd love to hear the answer on this. I need my right alt for language specific characters

**Reply by pifalken** [reply]
Any progress on this?

**Comment by zak-grivell** [comment]
You could use Karibiner Elements to rebind right alt to the hyper key and then use that in aerospace as a workaround
